### DISCUSSION TITLE
[Bug?]: Not able to uninstall Yarn version 3

### DISCUSSION THREAD

**Discussion by faizhameed** [yarnpkg/berry#5247] [Q&A]
### Self-service

- [ ] I'd be willing to implement a fix

### Describe the bug

I tried bunch of commands, there is no proper way to uninstall yarn v3 from the system. Nothing of this wirtten in documentation as well
I install this on ec2, i can't find a way to remove yarn globally
I tried 
``` 
sudo yum remove yarn
```
no luck, yarn still there

Im using Amaozn linux 2 in EC2 instance

### To reproduce

Try install yarn version 3+
Now try uninstall yarn using
```
sudo yum remove yarn
```

result
```
No Match for argument: yarn
No Packages marked for removal

```

### Environment

```shell
System:
    OS: Linux 5.10 Amazon Linux 2
    CPU: (1) x64 Intel(R) Xeon(R) CPU E5-2676 v3 @ 2.40GHz
  Binaries:
    Node: 16.19.0 - /tmp/xfs-d52b7e7f/node
    Yarn: 3.4.1 - /tmp/xfs-d52b7e7f/yarn
    npm: 8.19.3 - ~/.nvm/versions/node/v16.19.0/bin/npm
```

**Comment by RDIL** [comment]
Try running `which yarn` to determine where the binary is being provided from.

**Reply by faizhameed** [reply]
~/.yarn/bin/yarn

**Reply by faizhameed** [reply]
I can't see anything else

**Reply by faizhameed** [reply]
I now some how delete all the file related to yarn and reinstalled 1.22.19
